setup.py

```diff
@@ -7,7 +7,9 @@
 
 requirements = ['Click>=6.0',
                 'lxml',
-                'future']
+                'future',
+                'jsonschema',
+                'rpc-zigzag~=1.0.0']
```

Comment:
This pin is too tight.

```suggestion
                'rpc-zigzag~=1.0']
```